### PR TITLE
removed root of trust terminology fixes #395

### LINKED ISSUE #395: Removing root of trust terminology
It makes sense to remove the term 'root of trust' since the document doesn't define it. It might make sense to reference other industry specs that have relevant definitions such as NISTIR 8320A.

### PR DESCRIPTION
**Pull request by nedmsmith**
Reworded sections containing root of trust leaving a single parenthetical that associates the term loosely with an attesting environment that is implicitly trusted.

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
 compromised devices from getting access to the network and potentially
 harming others.
 
-Typically, solutions start with a specific component (called a root of trust) that is intended to
-provide trustworthy device identity and protected storage for measurements.
-The system components perform a series of measurements that may be
-signed via functions provided by a root of trust, considered as Evidence about present system components, such as hardware,
-firmware, BIOS, software, etc.
+Typically, a trustworthy solution starts with a specific component (sometimes referred to as the root of trust) that often
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:225]:
```suggestion
Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
```

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -430,15 +431,15 @@ of the TPM together with whatever component is feeding it the measurements.
...
-The root layer of an Attester includes at least one root of trust.
-In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
-Trust in the Attester's root of trust can be established in various ways as discussed in {{verifier}}.
+The bottom layer of an Attester has an Attesting Environment that is implicitly trusted, typically because it is designed to be immutable or difficult to modify by malicious code.
+In order to appraise Evidence generated by an Attester, the Verifier needs to trust various layers, including the bottom Attesting Environment.
+Trust in the Attester's layers, including the bottom layer, can be established in various ways as discussed in {{verifier}}.
 
-In layered attestation, a root of trust is the initial Attesting Environment.
-Claims can be collected from or about each layer.
+In layered attestation, an implicitly trusted Attesting Environment is the initial layer.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:438]: If you have endorsements, then it's not "implicitly" trusted.

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
...
 
-Typically, solutions start with a specific component (called a root of trust) that is intended to
-provide trustworthy device identity and protected storage for measurements.
-The system components perform a series of measurements that may be
-signed via functions provided by a root of trust, considered as Evidence about present system components, such as hardware,
-firmware, BIOS, software, etc.
+Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
+provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.
+The trusted components perform operations that help determine the trustworthiness of yet other components,
+such as collecting, protecting or signing measurements.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:228]:
```suggestion
such as by collecting, protecting or signing measurements.
```

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
...
+Measurements that have been signed by a trusted component is Evidence that either supports or contradicts a claim of trustworthiness.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:229]:
```suggestion
Measurements that have been signed by a trusted component comprises Evidence that either supports or contradicts a claim of trustworthiness.
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
...
-Typically, solutions start with a specific component (called a root of trust) that is intended to
-provide trustworthy device identity and protected storage for measurements.
-The system components perform a series of measurements that may be
-signed via functions provided by a root of trust, considered as Evidence about present system components, such as hardware,
-firmware, BIOS, software, etc.
+Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
+provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.
+The trusted components perform operations that help determine the trustworthiness of yet other components,
+such as by collecting, protecting or signing measurements.
+Measurements that have been signed by a trusted component comprises Evidence that either supports or contradicts a claim of trustworthiness.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:229]:
```suggestion
Measurements that have been signed by such components comprise Evidence that either supports or contradicts a claim of trustworthiness.
```

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
...
 
-Typically, solutions start with a specific component (called a root of trust) that is intended to
-provide trustworthy device identity and protected storage for measurements.
-The system components perform a series of measurements that may be
-signed via functions provided by a root of trust, considered as Evidence about present system components, such as hardware,
-firmware, BIOS, software, etc.
+Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
+provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.
+Such components perform operations that help determine the trustworthiness of yet other components,
+such as by collecting, protecting or signing measurements.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:228]:
```suggestion
Such components perform operations that help determine the trustworthiness of yet other components,
by collecting, protecting or signing measurements.
```

File: draft-ietf-rats-architecture.md [hunk 7]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
...
-provide trustworthy device identity and protected storage for measurements.
-The system components perform a series of measurements that may be
-signed via functions provided by a root of trust, considered as Evidence about present system components, such as hardware,
-firmware, BIOS, software, etc.
+Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
+provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.
+Such components perform operations that help determine the trustworthiness of yet other components,
+by collecting, protecting or signing measurements.
+Measurements that have been signed by such components comprise Evidence that either supports or contradicts a claim of trustworthiness.
+Measurements can describe a variety of component attributes, such as hardware, firmware, BIOS, software, etc.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:230]:
```suggestion
Measurements can describe a variety of attributes of system components, such as hardware, firmware, BIOS, software, etc.
```

File: draft-ietf-rats-architecture.md [hunk 8]
```diff
@@ -430,15 +431,15 @@ of the TPM together with whatever component is feeding it the measurements.
 By definition, the Attester role generates Evidence.
 An Attester may consist of one or
 more nested environments (layers).
-The root layer of an Attester includes at least one root of trust.
-In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
-Trust in the Attester's root of trust can be established in various ways as discussed in {{verifier}}.
+The bottom layer of an Attester has an Attesting Environment that is implicitly trusted, typically because it is designed to be immutable or difficult to modify by malicious code.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:434]:
```suggestion
The bottom layer of an Attester has an Attesting Environment that is typically designed to be immutable or difficult to modify by malicious code.
```

File: draft-ietf-rats-architecture.md [hunk 9]
```diff
@@ -430,15 +431,15 @@ of the TPM together with whatever component is feeding it the measurements.
...
+Trust in the Attester's layers, including the bottom layer, can be established in various ways as discussed in {{verifier}}.
 
-In layered attestation, a root of trust is the initial Attesting Environment.
-Claims can be collected from or about each layer.
+In layered attestation, an implicitly trusted Attesting Environment is the initial layer.
+Claims can be collected from or about each layer beginning with an initial layer.
 The corresponding Claims can be structured in a nested fashion that reflects the nesting of the Attester's layers.
 Normally, Claims are not self-asserted, rather a previous layer acts as the Attesting Environment for the next layer.
-Claims about a root of trust typically are asserted by an Endorser.
+Claims about an implicitly trusted layer typically are asserted by an Endorser.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:442]:
```suggestion
Claims about an initial layer typically are asserted by an Endorser.
```

File: draft-ietf-rats-architecture.md [hunk 10]
```diff
@@ -430,15 +431,15 @@ of the TPM together with whatever component is feeding it the measurements.
...
-In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
-Trust in the Attester's root of trust can be established in various ways as discussed in {{verifier}}.
+The bottom layer of an Attester has an Attesting Environment that is typically designed to be immutable or difficult to modify by malicious code.
+In order to appraise Evidence generated by an Attester, the Verifier needs to trust various layers, including the bottom Attesting Environment.
+Trust in the Attester's layers, including the bottom layer, can be established in various ways as discussed in {{verifier}}.
 
-In layered attestation, a root of trust is the initial Attesting Environment.
-Claims can be collected from or about each layer.
+In layered attestation, an implicitly trusted Attesting Environment is the initial layer.
+Claims can be collected from or about each layer beginning with an initial layer.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:439]:
```suggestion
In layered attestation, Claims can be collected from or about each layer beginning with an initial layer.
```

File: draft-ietf-rats-architecture.md [hunk 11]
```diff
@@ -977,7 +978,7 @@ to convey Evidence that is otherwise unprotected (e.g., not signed). Appropriate
 
    1. The key material used to authenticate and integrity protect the conveyance channel is trusted by the Verifier to speak for the Attesting Environment(s) that collected Claims about the Target Environment(s).
    2. All unprotected Evidence that is conveyed is supplied exclusively by the Attesting Environment that has the key material that protects the conveyance channel
-   3. The root of trust protects both the conveyance channel key material and the Attesting Environment with equivalent strength protections.
+   3. A trusted environment protects the conveyance channel key material which may depend on other Attesting Environments with equivalent strength protections.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:980]:
```suggestion
   3. A trusted environment protects the conveyance channel's key material which may depend on other Attesting Environments with equivalent strength protections.
```

File: draft-ietf-rats-architecture.md [hunk 12]
```diff
@@ -222,11 +222,12 @@ Remote attestation is desired to prevent vulnerable or
...
-Typically, solutions start with a specific component (called a root of trust) that is intended to
-provide trustworthy device identity and protected storage for measurements.
-The system components perform a series of measurements that may be
-signed via functions provided by a root of trust, considered as Evidence about present system components, such as hardware,
-firmware, BIOS, software, etc.
+Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
+provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.
+Such components perform operations that help determine the trustworthiness of yet other components,
+by collecting, protecting or signing measurements.
+Measurements that have been signed by such components comprise Evidence that either supports or contradicts a claim of trustworthiness.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:229]:
```suggestion
Measurements that have been signed by such components comprise Evidence that when evaluated either supports or refutes a claim of trustworthiness.
```

**Pull request merged**